cpp/CMakeLists.txt

```diff
@@ -144,7 +144,7 @@ rapids_find_package(
 )
 
 # add third party dependencies using CPM
-rapids_cpm_init()
+rapids_cpm_init(OVERRIDE ${CMAKE_CURRENT_SOURCE_DIR}/cmake/versions.json)
```

Comment:
I don't think cuDF currently has a `versions.json`?

Reply:
You're right. Should it ?   @robertmaynard do you think we should have versions.json for each rapids lib to better track versions of cuco, libcudacxx, thrust, etc ?

Reply:
We rely on all of RAPIDS sharing common versions of those core dependencies. There are some CMake bits that could change from package to package, but we usually source those core dependencies from `rapids-cmake`: https://github.com/rapidsai/rapids-cmake/blob/branch-22.12/rapids-cmake/cpm/versions.json

Reply:
Bradley is correct, but my guess is that this line was changed in DFLW because there was a specific package for which a newer or patched version was required. The question would be whether that was a bugfix that we need to upstream (in which case we should put that in rapids-cmake) or if it's specific to DLFW.

Reply:
Sorry for the late response. But I agree with Bradley and Vyas we should drop the versions.json and rely on the versions.json that exists in rapids-cmake